charts/wasp-thing-service-v2/templates/deployment.yaml

```diff
@@ -0,0 +1,234 @@
+              then
+                echo "Database already exists doing nothing";
+                exit 0;
+              fi;
+              createuser $CONN_FLAGS $DB_USERNAME;
+              PSQL_ESC=$(echo $DB_PASSWORD | sed 's/./_/g');
+              psql $CONN_FLAGS -c "ALTER ROLE $DB_USERNAME WITH PASSWORD \$$$PSQL_ESC\$$$DB_PASSWORD\$$$PSQL_ESC\$";
+              createdb $CONN_FLAGS -O $DB_USERNAME $DB_NAME;
+        {{- end }}
+        {{- if .Values.initDbMigrate.enable }}
+        - name: database-migrate
+          image: {{ template "wasp-thing-service.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command: ['npx', 'knex']
+          {{- if .Values.initDbMigrate.args }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.initDbMigrate.args "context" $) | nindent 12 }}
+          {{- end }}
+          env:
+            - name: NODE_ENV
+              value: {{ .Values.initDbMigrate.environment }}
+            - name: DB_HOST
+              value: {{ template "wasp-thing-service.databaseHost" . }}
+            - name: DB_PORT
+              value: {{ template "wasp-thing-service.databasePort" . }}
+            - name: DB_NAME
+              value: {{ template "wasp-thing-service.databaseName" . }}
+            - name: DB_USERNAME
+              value: {{ template "wasp-thing-service.databaseUser" . }}
+            - name: DB_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "wasp-thing-service.databaseSecretName" . }}
+                  key: {{ include "wasp-thing-service.databaseSecretPasswordKey" . }}
+        {{- end }}
+        {{- if .Values.initContainers }}
+          {{- include "common.tplvalues.render" (dict "value" .Values.initContainers "context" $) | nindent 8 }}
+        {{- end }}
+      {{- end }}
+      containers:
+        - name: wasp-thing-service
+          image: {{ template "wasp-thing-service.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          {{- if .Values.containerSecurityContext.enabled }}
+          securityContext: {{- omit .Values.containerSecurityContext "enabled" | toYaml | nindent 12 }}
+          {{- end }}
+          {{- if .Values.diagnosticMode.enabled }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.command "context" $) | nindent 12 }}
+          {{- else if .Values.command }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.command "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.diagnosticMode.enabled }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.args "context" $) | nindent 12 }}
+          {{- else if .Values.args }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.args "context" $) | nindent 12 }}
```

Comment:
This could probably be squashed into a couple of includes for `command` and another for `args` rather than the duplication?

Reply:
again it's taken from the bitnami template. This is probably the smallest version of it without writing our own template library